docs/data/joy/customization/using-css-variables/using-css-variables.md

```diff
@@ -1,81 +1,97 @@
 # Using CSS variables
 
-<p class="description">Learn how to use CSS variables to customize Joy UI's components.</p>
+<p class="description">Learn how to use CSS variables to customize Joy UI components.</p>
 
-## Theme object
+## Introduction
 
-The `CssVarsProvider` reads the theme input (or the default theme) and creates the CSS variables according to the theme structure.
-It also creates an object that refers to the generated CSS variables under `theme.vars` so that you can use them from the JavaScript theme object.
+CSS variables are created by wrapping your app with the CssVarsProvider and using the `theme.vars` notation, allowing for an efficient customization experience.
 
-The `theme.vars` is available in all styling APIs that Joy UI offers:
+```jsx
+import { CssVarsProvider } from '@mui/joy/styles';
+
+function App() {
+  return <CssVarsProvider>...</CssVarsProvider>;
+}
+```
 
-1. The `styled` function
+## Usage
+
+### Styling APIs
+
+Use the `theme.vars` notation with any styling APIs supported by Joy UI:
```

Comment:
I don't know if we have a preferred style for this, but I like adding `.*` to indicate that it's incomplete/open-ended.

```suggestion
Use the `theme.vars.*` notation with any styling APIs supported by Joy UI:
```